.github/workflows/import.yml

```diff
@@ -31,11 +34,21 @@ jobs:
           script: |
             const tag = '${{ github.event.inputs.tag }}';
             const sha = '${{ steps.import-release.outputs.sha }}';
+            const mainRepoOwner = '${{ env.MAIN_REPO_OWNER }}';
+            const mainRepoRepo = '${{ env.MAIN_REPO_REPO }}';
+
+            const { data } = await github.repos.getReleaseByTag({
+              owner: mainRepoOwner,
+              repo: mainRepoRepo,
+              tag,
+            });
+
             await github.repos.createRelease({
               owner: context.repo.owner,
               repo: context.repo.repo,
               target_commitish: sha,
               tag_name: tag,
               name: tag,
-              body: `Mirror of https://github.com/BranchMetrics/ios-branch-deep-linking-attribution/releases/${tag}`,
+              body: `Mirror of https://github.com/${mainRepoOwner}/${mainRepoRepo}/releases/${tag}`,
+              prerelease: data.prelease,
```

Comment:
This actually wasn't working when creating test releases in the main repo. Might be necessary to change the flag after creating. For now, need to be careful about using this for testing.